docs/content/management-bundles.md

```diff
@@ -419,6 +459,103 @@ bundle.RegisterSigner("custom", &CustomSigner{})
 bundle.RegisterVerifier("custom", &CustomVerifier{})
 ```
 
+### Delta Bundles
+
```

Comment:
Do we need to say any bundle up until now is called a "snapshot bundle" from hereon? It's new terminology, I think...?

Reply:
I've added this line in the intro to Delta Bundles. Do you think this should suffice or we need to provide more clarification somewhere else?
```
A regular _snapshot_ bundle represents the entirety of OPA’s policy and data cache. When a new _snapshot_ bundle is
downloaded, OPA will erase and overwrite all the policy and data in its cache before activating the new bundle. We can
optionally scope the bundle to a subset of OPA’s policy and data cache by defining the `roots` in the bundle's manifest.
```